### PR TITLE
Support for $tags

### PR DESCRIPTION
**Pull request by danielmarbach**
@Particular/nservicebus-maintainers please review

Heads up

@Particular/transport-maintainers @Particular/servicecontrol-maintainers @Particular/serviceinsight-maintainers @Particular/servicepulse-maintainers 

This is just a convenience thing. You don't have to pull it into the downstreams. I just wanted to make sure that you guys know about this PR and the feature it enables. All the info is contained in the README.md

### CODE REVIEW

**Comment by timbussmann** [comment]
what's the intent on this? It kinda seems like we should change the tags to something more appropriate on the downstream repositories?

**Comment by danielmarbach** [comment]
We use the tags of NSB Core in all downstreams

**Comment by danielmarbach** [comment]
So although you might be right, that is currently not what we do. This PR just makes it possible to use the standard tags instead of repeating it all over again.

**Comment by bording** [comment]
> We use the tags of NSB Core in all downstreams

Right now in the RabbitMQ nuspec file, the tags do match the set listed in this PR. Using that list though introduces an oddity in that the RabbitMQ transport mentions msmq, but doesn't mention RabbitMQ itself.

**Comment by danielmarbach** [comment]
Msmq is probably there for legacy reason. We could just remove msmq form the standard tags. You can always opt out from not using those.

**Comment by danielmarbach** [comment]
Removed msmq from standard tags. @Particular/nservicebus-maintainers please review and merge

File: src/NuGetPackager/PackageCreator.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
     class PackageCreator : IPropertyProvider
     {
-        readonly Dictionary<string, Func<string>> propertyAssignments; 
+        readonly Dictionary<string, Func<string>> propertyAssignments;
```
Comment (andreasohlund) [inline src/NuGetPackager/PackageCreator.cs]: Fertile ground for @SimonCropp 's "readonly" inquisition :)

the_spanish_inquisition_zpsd96e40e9

Comment (SimonCropp) [inline src/NuGetPackager/PackageCreator.cs]: are the irony. marking a mutable collection with readonly

**Comment by timbussmann** [comment]
> Removed msmq from standard tags

so we even have to override the standard tags in the core? It still looks like there isn't a single package which should use the standard tags alone?

File: src/NuGetPackager/PackageCreator.cs (hunk 1 above)
Comment (danielmarbach) [inline src/NuGetPackager/PackageCreator.cs]: By the way this was there before ;) Happy to fix it in another PR.

**Comment by danielmarbach** [comment]
@timbussmann Sorry this leads to nowhere. The fact is, all packages are currently using the standard tags. This PR allows replacing the copy pasted standard tags with the $tag placeholder. Nothing more, nothing less. I think it is worth taking this in and then discuss the "larger" issues in another focused discussion.

**Comment by danielmarbach** [comment]
Pulling in.

**Pull request merged**